### ISSUE TITLE
Pin is not in center in the minimap

### ISSUE THREAD

**Issue by diba78** [xbgmsharp/piwigo-openstreetmap#173]
Hi togehther,

i have a problem with the minimap on the photo site. the location pin is not in the center and one part of the minimap is gray.

i attached in example.

Is there any config task that can resolve this ?

Best regards.

Dirk
![Minimap_problem](https://user-images.githubusercontent.com/44620652/64376288-38b27700-d028-11e9-820e-77f6c3d0e1e8.JPG)

**Comment by hummeruli** [comment]
It was different in resolution. I've also a grey field on the right.

Uli

**Comment by travelsignposts** [comment]
Yes, I've been having exactly this issue since the end of 2022, but have as yet not found a solution.  Currently the map appears OK and with pin centred on my iPhone, but on the desktop it appears as shown above, displaced but with a narrower map. However, if I resize the window in any way the map appears correctly with the pin centred. If I reload this corrected page, once again the map and pin is displaced as earlier. As before, a slight resize of the window (Chrome or Edge) produces the correct version. So it's a common experience.
